api/src/main/java/org/eclipse/microprofile/config/spi/ConfigBuilder.java

```diff
@@ -42,77 +42,83 @@
 @org.osgi.annotation.versioning.ProviderType
 public interface ConfigBuilder {
     /**
-     * Add the default config sources appearing on the builder's classpath
-     * including:
-     * <ol>
-     * <li>System properties</li>
-     * <li>Environment properties</li>
-     * <li>/META-INF/microprofile-config.properties</li>
-     * </ol>
+     * Add the <a href="ConfigSource.html#default_config_sources"><em>default configuration sources</em></a>
```

Comment:
Once this is in the javadoc, I don't think the reference link will work though.

Reply:
It should work: the referenced class is in the same package as this class. This pattern is used extensively within the JDK.

Reply:
ah. ok. I thought it linked to the spec.